### PR TITLE
Fix/423 design system update change the fields label

### LINKED ISSUE #423: Design system update - Change the fields' label
Design system update - Change the fields' label (text/area/number/search/checkbox/radio button, etc.) to use "primary-text" instead of "secondary-text"

Check with @mmarsoleta for more info.

### PR DESCRIPTION
**Pull request by mgaseta**
# Description

Updated theme package and other style fixes.

Fixes #423

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Only visual changes, so only tested manually.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-529-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-529-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-529-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Comment by craigyu** [comment]
Hello, PR #532 should help with the Javascript CI failure.

File: frontend/src/components/FormReview/styles.scss [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
+  .form-review-title-section {
+    height: 2.625rem;
+    margin: 2rem;
+  }
+
+  .item-section {
+    padding-left: 1rem;
+  }
+
+  .form-item{
```
Comment (RMCampos) [inline frontend/src/components/FormReview/styles.scss:25]: suggestion (non-blocking): add space between classname and opening brackets.

File: frontend/src/components/FormReview/styles.scss [hunk 2]
```diff
@@ -1,44 +1,48 @@
...
+    padding-top:  2.0rem
+  }
+
+  .item-description-section {
+    @include type.type-style('body-compact-01');
+    color: var(--#{vars.$bcgov-prefix}-text-secondary);
+    padding-top: 0.25rem;
+  }
+
+  .form-review-empty-section .empty-section-container{
```
Comment (RMCampos) [inline frontend/src/components/FormReview/styles.scss:36]: suggestion (non-blocking): add space between classname and opening brackets.

**Review by RMCampos — APPROVED** [review]
Great job! 🥇 Some non-blocking suggestions only. LGTM

File: frontend/src/components/FormReview/styles.scss (hunk 1 above)
Comment (mgaseta) [reply frontend/src/components/FormReview/styles.scss]: Nice suggestion, fixed: https://github.com/bcgov/nr-spar/pull/529/commits/bfcc4dd08e8536a2c0fc88f445d7e1cc99687252

File: frontend/src/components/FormReview/styles.scss (hunk 2 above)
Comment (mgaseta) [reply frontend/src/components/FormReview/styles.scss]: Adjusted: https://github.com/bcgov/nr-spar/pull/529/commits/bfcc4dd08e8536a2c0fc88f445d7e1cc99687252

**Review by craigyu — APPROVED** [review]
💯 A great example on how using the theme package is saving time for developers <sub>/s</sub>

**Pull request merged**